src/styles/index.scss

```diff
@@ -1,3 +1,100 @@
+@import 'fonts';
+@import 'stars';
+@import 'variables';
+
 body {
   margin: 0;
+  padding: 0;
+  font-family: $fontRoboto;
+  font-size: 12px;
+  box-sizing: border-box;
+}
+
+.product-card {
+  box-sizing: border-box;
+  width: 200px;
+  height: 408px;
+  padding: 16px;
+  padding-top: 32px;
```

Comment:
You have already declared padding property on line 17. It's better to combine them into one declaration to avoid overwriting and improve code readability. You can write it like this: `padding: 32px 16px 16px;`.